pkg/remediator/queue/queue.go

```diff
@@ -44,11 +46,15 @@ func GVKNNOf(obj client.Object) GVKNN {
 	}
 }
 
+// ErrShutdown is returned by `ObjectQueue.Get` if the queue is shut down
+// before an object is available/added.
+var ErrShutdown = errors.New("object queue shutting down")
```

Comment:
Maybe define this as a function rather than a `var` to enforce that this doesn't get changed?

Reply:
Yeah, that's potentially a concern, but this method has precedent in both the std libs and k8s libs and makes it easy to do an == comparison.